test-programs/exec-off-leader/src/main.rs

```diff
@@ -1,7 +1,11 @@
 use std::os::unix::process::CommandExt;
 use std::process::Command;
+use std::thread::{sleep, spawn};
+use std::time::Duration;
 
 fn main() {
     let mut cmd = Command::new("/bin/true");
-    std::thread::spawn(move || cmd.exec()).join().unwrap();
+    spawn(|| sleep(Duration::from_secs(60)));
+    sleep(Duration::from_millis(100));  // Hack, sorry.
```

Comment:
We can remove this after adding support for event assertion modulo interleaving.